packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -33,6 +33,8 @@ type DocTypes =
   | DocType<'Tutorial', 'tutorial'>
   | DocType<'Docs', 'docs'>
   | DocType<'API', 'api'>
+  | DocType<'Video', 'video'>
+  | DocType<'MDN', 'other'>
```

Comment:
Do the other options also need to be added?

E.g.: "WebKit", "Web.dev", "StackBlitz"

Reply:
idk why i really included the first part of the type. It should really be `<string, 'tutorial'|'docs'|'api'|'video'|'other'>` the first part doesn't _really_ need to be there